Patch by MSP-Greg
changes STDERR output to STDOUT, keeps Powershell happy
diff --git a/tool/transcode-tblgen.rb b/tool/transcode-tblgen.rb
index 156b2de197..7f87fce661 100644
--- a/tool/transcode-tblgen.rb
+++ b/tool/transcode-tblgen.rb
@@ -724,7 +724,7 @@ def citrus_decode_mapsrc(ces, csid, mapsrcs)
     path = File.join(*path)
     path << ".src"
     path[path.rindex('/')] = '%'
-    STDERR.puts 'load mapsrc %s' % path if VERBOSE_MODE
+    STDOUT.puts 'load mapsrc %s' % path if VERBOSE_MODE
     open(path, 'rb') do |f|
       f.each_line do |l|
         break if /^BEGIN_MAP/ =~ l
@@ -824,9 +824,9 @@ def transcode_compile_tree(name, from, map, valid_encoding)
 def transcode_tbl_only(from, to, map, valid_encoding=UnspecifiedValidEncoding)
   if VERBOSE_MODE
     if from.empty? || to.empty?
-      STDERR.puts "converter for #{from.empty? ? to : from}"
+      STDOUT.puts "converter for #{from.empty? ? to : from}"
     else
-      STDERR.puts "converter from #{from} to #{to}"
+      STDOUT.puts "converter from #{from} to #{to}"
     end
   end
   id_from = from.tr('^0-9A-Za-z', '_')
@@ -905,7 +905,7 @@ def transcode_tblgen(from, to, map, valid_encoding=UnspecifiedValidEncoding,
 end
 
 def transcode_generate_node(am, name_hint=nil)
-  STDERR.puts "converter for #{name_hint}" if VERBOSE_MODE
+  STDOUT.puts "converter for #{name_hint}" if VERBOSE_MODE
   am.gennode(TRANSCODE_GENERATED_BYTES_CODE, TRANSCODE_GENERATED_WORDS_CODE, name_hint)
   ''
 end
@@ -1066,14 +1066,14 @@ def make_signature(filename, src)
     if old_signature == chk_signature
       now = Time.now
       File.utime(now, now, output_filename)
-      STDERR.puts "already up-to-date: #{output_filename}" if VERBOSE_MODE
+      STDOUT.puts "already up-to-date: #{output_filename}" if VERBOSE_MODE
       exit
     end
   end
 
   if VERBOSE_MODE
     if output_filename
-      STDERR.puts "generating #{output_filename} ..."
+      STDOUT.puts "generating #{output_filename} ..."
     end
   end
 
@@ -1111,7 +1111,7 @@ def make_signature(filename, src)
     File.rename(new_filename, output_filename)
     tms = Process.times
     elapsed = Time.now - start_time
-    STDERR.puts "done.  (#{'%.2f' % tms.utime}user #{'%.2f' % tms.stime}system #{'%.2f' % elapsed}elapsed)" if VERBOSE_MODE
+    STDOUT.puts "done.  (#{'%.2f' % tms.utime}user #{'%.2f' % tms.stime}system #{'%.2f' % elapsed}elapsed)" if VERBOSE_MODE
   else
     print result
   end
